use ?\s instead of ?(whitespace)

conao3 opened the pull request:
`? ` and `?\ ` and `?\s` are return same value.
`?\s` is easier to read and understand.

```emacs-lisp
(list ?  ?\  ?\s)
;;=> (32 32 32)
```
ref: https://www.gnu.org/software/emacs/manual/html_mono/elisp.html#Basic-Char-Syntax

ikazuhiro commented:
Emacs 21 and earlier don't support that syntax.
Please send PR to lexical-binding branch.

conao3 commented:
OK, I changed the base branch.

conao3 committed:
use ?\s instead of ?(whitespace)

? and ?\ and ?\s are return same value.
?\s is easier to read and understand.

(list ?  ?\  ?\s)
;;=> (32 32 32)

conao3 commented:
Resolve conflict.

conao3 commented:
Reindent under `(setq indent-tabs-mode t)`.

ikazuhiro commented:
Merged, thank you!

conao3 commented:
Thanks!
